docs/sources/alerting/fundamentals/alert-rules/queries-conditions.md

```diff
@@ -111,6 +111,12 @@ The threshold expression allows the comparison between two single values. Availa
 - **Is below**: `$B < 3`
 - **Is within range**: `$A > 0 AND $A < 10`
 - **Is outside range**: `$B < 0 OR $B > 100`
+- **Equal**: `$A == 2`
+- **Not equal**: `$B =! 4`
+- **Greater or equal**: `$A >= 8`
+- **Less or equal**: `$B >= 16`
+- **Is within range, inclusive**: `$A <= 32 || $A >= 64`
```

Comment:
```suggestion
- **Is within range, inclusive**: `$A >= 0 AND $A <= 10`
```

Reply:
using `0` and `10` to be consistent with the same range used in the `Is within range` example